modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -227,18 +227,22 @@ export default class Tile3DLayer extends CompositeLayer {
 
   _makeSimpleMeshLayer(tileHeader, oldLayer) {
     const content = tileHeader.content;
-    const {attributes, modelMatrix, cartographicOrigin, texture} = content;
-    const {positions, normals, texCoords} = attributes;
+    const {modelMatrix, cartographicOrigin, texture} = content;
+    const attributes = {};
+    attributes.positions = content.attributes.positions;
+    attributes.positions.value = new Float32Array(attributes.positions.value);
+    if (content.attributes.normals) {
+      attributes.normals = content.attributes.normals;
+    }
+    if (content.attributes.texCoords) {
+      attributes.texCoords = content.attributes.texCoords;
+    }
 
     const geometry =
       (oldLayer && oldLayer.props.mesh) ||
       new Geometry({
         drawMode: GL.TRIANGLES,
-        attributes: {
-          positions: {...positions, value: new Float32Array(positions.value)},
-          normals,
-          texCoords
```

Comment:
You can simply fix this by spreading the original attributes here:

```js
attributes: {
  ...content.attributes,
  positions: {...positions, value: new Float32Array(positions.value)}
}
```

Reply:
I tried but the geometry can have other attributes with Float32Array (eg `featureIds`)

Reply:
Just curious: Does that cause a problem? Are they used?

Reply:
I mean full attributes map contain some unexpected attributes in this context. I think we have to get each attribute explicitly as before.